cloud/kubernetes/helm/yugabyte/templates/service.yaml

```diff
@@ -196,6 +197,7 @@ spec:
           - "--tserver_master_addrs={{range $index := until (int ($root.Values.replicas.master))}}{{if ne $index 0}},{{end}}yb-master-{{ $index }}.yb-masters.$(NAMESPACE):7100{{end}}"
           - "--tserver_master_replication_factor={{ $root.Values.replicas.master }}"
```

Comment:
I also looked at the discussion in ENG-4215 and it seems the best way moving forward is to remove the flag for the tserver_master_replication_factor (it'll default to 0)